.github/renovate.json

```diff
@@ -14,17 +15,15 @@
      ],
      "stabilityDays":0
   },
+  "separateMajorMinor":false,
   "packageRules": [
     {
-      "matchDepTypes": ["module"],
-      "groupName": "TF modules",
-      "separateMajorMinor":false
+      "matchPaths": [".github/**"],
```

Comment:
I may have misunderstood the config but wouldn't this prevent modules from being bumped?

Reply:
Ah makes sense!